src/classes/Recipe.js

```diff
@@ -0,0 +1,35 @@
+class Recipe {
+    constructor(recipeDetails) {
+        this.id = recipeDetails.id;
+        this.name = recipeDetails.name;
+        this.image = recipeDetails.image;
+        this.portions = recipeDetails.portions;
+        this.instructions = recipeDetails.instructions;
+        this.tags = recipeDetails.tags;
+
+    }
+    showIngredientsNeeded() {
```

Comment:
I'm thinking we should rename this `getPortionNames`. What do you think?

Reply:
Sounds good to me, I made that adjustment.